sparse/src/KokkosSparse_Utils.hpp

```diff
@@ -933,25 +934,13 @@ struct ColorChecker {
   }
 };
 
-/**
- * \brief given a graph and a coloring function returns true or false if
- distance-1 coloring is valid or not.
- * \param num_rows: num rows in input graph
- * \param num_cols: num cols in input graph
- * \param xadj: row pointers of the input graph
- * \param adj: column indices of the input graph
- * \param t_xadj: output, the row indices of the output graph. MUST BE
- INITIALIZED WITH ZEROES.
-
- * \param vector_size: suggested vector size, optional. if -1, kernel will
- decide.
- * \param suggested_team_size: suggested team size, optional. if -1, kernel will
- decide.
- * \param team_work_chunk_size: suggested work size of a team, optional. if -1,
- kernel will decide.
- * \param use_dynamic_scheduling: whether to use dynamic scheduling. Default is
- true.
- */
+/// \brief given a graph and a coloring function returns true or false if
+///        distance-1 coloring is valid or not.
+///
+/// \param num_rows num rows in input graph
+/// \param xadj     row pointers of the input graph
+/// \param adj      column indices of the input graphw
+/// \param v_colors
```

Comment:
```suggestion
/// \param v_colors The colors at each vertex in the graph.
```
@brian-kelley: Is this parameter description correct?